package.json

```diff
@@ -28,12 +28,12 @@
     "csso": "~3.5.0",
     "filesize": "^3.5.11",
     "minimist": "^1.2.0",
-    "puppeteer": "^1.4.0"
+    "puppeteer": "1.6.0"
```

Comment:
cc @peterbe renovate doesn't update puppeteer, because it has un-exact version e.g. `^`.

Reply:
Renovate does the right thing. The only reason that line should change is if you and I decide that people should upgrade puppeteer. Basically, minimalcss works with *at least* 1.4.0. 

If we have a feature within that depends on a more recent version of puppeteer, we should change that line to `^1.6.0` until 1.6.0 is no longer good enough.

Reply:
not sure what you suggest. Leave it as is? or fix to 1.4.0?

Reply:
Leave it as `^1.4.0` unless you think we *should* upgrade `minimalcss` because 1.6.0 is much better and we should "force" that upgrade.

Reply:
no it's fine. Let's leave it